src/main/java/org/epam/core/utils/WaitUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.epam.core.utils;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.awaitility.core.ConditionFactory;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class WaitUtils {
+
+    public static final int ONE_SECOND = 1;
+    public static final int THIRTY_SECONDS = 30;
+
+    public static ConditionFactory kindlyWait(final long seconds) {
```

Comment:
May I know why do you have here long and int in #kindlyWait(final int duration, final TimeUnit timeUnit)?

Reply:
Thanks for comment! Fixed.